modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -1,63 +1,93 @@
 = Back up and Restore Runtime Fabric
 
-To preserve applications in the event of system failure, ensure that Runtime Fabric is automatically backed up. 
-MuleSoft highly recommends configuring hourly backups. Store your backup archive on external storage outside 
-the Runtime Fabric cluster.
+To preserve applications in the event of system failure, confirm that Runtime Fabric is automatically backed up. When scheduling backups, follow best practices by configuring hourly backups. Store your backup archive on external storage outside the Runtime Fabric cluster.
+
+The state of Runtime Fabric is distributed across the Kubernetes cluster. Use the following procedures to back up and restore the required state necessary for all deployed applications, management services, and their configurations. Individual node backup and restore operations are not supported. 
```

Comment:
`Individual node backup and restore operations are not supported` we can perhaps clarify this better? Perhaps VM snapshot backup/restore is not supported?